redfish-client/etc/redfish-client.conf

```diff
@@ -0,0 +1,2 @@
+[redfish-client]
+templates_path = /usr/share/redfish-client/templates
```

Comment:
We need to use a macro here to support multiple possibilities for different env.